opcpa-tpr-config/opcpa-tpr-config.py

```diff
@@ -0,0 +1,164 @@
+import yaml
+
+from functools import partial
+
+from qtpy.QtWidgets import QGridLayout
+
+from ophyd import EpicsSignal
+
+from pydm import Display
+from pydm.widgets import PyDMLabel, PyDMPushButton, PyDMNTTable
+
+
+class App(Display):
+    def __init__(self, parent=None, args=None, macros=None):
+        # print(f'args={args}, macros={macros}')
+
+        # Read in config file
+        # TODO add argparse, make this an arg
```

Comment:
TODO Reminder.  If you want to leave this for later that's fine, but you could create the argparse in __main__.py and pass arguments to PyDMApplication.  (I'm still familiarizing myself with the best way to do things here myself)